src/test/scripts/README.md

```diff
@@ -0,0 +1,25 @@
+
+# check-merge-to-enterprise Job
+
+When you open a PR on community, if it creates a conflict with enterprise-master, the check-merge-to-enterprise will fail. Say your branch name is `X`, we will refer to `X` on community as `community/X` and on enterprise as `enterprise/X`. In order to make the job pass:
+
+- You first need to get approval for `community/X`. Only follow the next steps after you are about to merge the branch to community master. Otherwise, the `check-merge-to-enterprise` check might falsely pass. (For example when you added new commits to `community/X` but forgot to update `enterprise/X`).
+- You need to create the same branch, `X` locally on enterprise side (don't push yet), and merge `enterprise-master` to `enterprise/X`. Solve any conflicts, on enterprise repository:
```

Comment:
I think create here is a bit too loosely defined, we don't want people to copy their code changes but have the exact same git commits.

Instead I think wording like synchronise `community/X` to `enterprise/X`, and explain this can be achieved by pushing your community branch to enterprise